conans/util/dates.py

```diff
@@ -31,14 +31,18 @@ def timestamp_to_str(timestamp):
 
 
 def timedelta_from_text(interval):
-    match = re.search(r"(\d+)([mhd])", interval)
+    match = re.search(r"(\d+)([smhdw])", interval)
     try:
         value, unit = match.group(1), match.group(2)
-        if unit == 'm':
+        if unit == 's':
+            return datetime.timedelta(seconds=float(value))
+        elif unit == 'm':
             return datetime.timedelta(minutes=float(value))
         elif unit == 'h':
             return datetime.timedelta(hours=float(value))
-        else:
+        elif unit == 'd':
```

Comment:
It may be a breaking change. Default was `days`, now we no longer accept implicit values. I don't think it's a good idea keeping implicit support, we would need read docs to understand what's the behavior.

Reply:
In the docs: 

> The config_install_interval variable starts a time scheduler which runs conan config install according the time interval configured. It only accepts the follow time intervals: minutes, hours and days.

It is lacking some details. It should also have been labeled as experimental, this could be a problem, I'd probably keep the default with a big warning.